src/drive/web/modules/views/OnlyOffice/Toolbar/ReadOnly.jsx

```diff
@@ -1,23 +1,38 @@
 import React from 'react'
 
 import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import LockIcon from 'cozy-ui/transpiled/react/Icons/Lock'
 import Tooltip from 'cozy-ui/transpiled/react/Tooltip'
 
 const ReadOnly = () => {
   const { t } = useI18n()
+  const { isMobile } = useBreakpoints()
+
+  if (isMobile)
+    return (
+      <div className="u-flex u-mr-1">
+        <Icon
+          data-testid="onlyoffice-readonly-icon"
```

Comment:
data-testid shouldn't be the same as the next one here https://github.com/cozy/cozy-drive/pull/2360/commits/422bd9c533572491fa2c051018d781f0e080e50f#diff-b4bff9eda39a0fe525b252406cb30397dd0469214ee6a2465224f8212e3ae16dR29

because when you do your test, you can not be sure if the right one is displayed or not.